products/BellHybrid/apps/application-bell-background-sounds/presenter/BGSoundsProgressPresenter.cpp

```diff
@@ -92,4 +92,8 @@ namespace app::bgSounds
     {
         return player.isPaused();
     }
+    void BGSoundsProgressPresenter::onBeforeShow()
+    {
+        getView()->setTimeFormat(timeModel->getTimeFormat());
```

Comment:
is this change related to this PR? What does it cause?

Reply:
Actually, this PR contains BH-1191 and BH-1204. These changes are related to BH-1204.

Reply:
It fixes incorrect 24/12 time format being displayed in bg sounds app :)